test-docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+echo "Building Docker image..."
+docker build -t culero-api .
+
+echo "Running Docker container..."
+docker run --rm -d \
```

Comment:
before running this command, you should also issue `docker compose up -d` since that will boot up the necessary containers.